src/services/known_tokens.ts

```diff
@@ -0,0 +1,44 @@
+import { WETH_TOKEN_SYMBOL } from '../common/constants';
+import { KNOWN_TOKENS_META_DATA } from '../common/tokens_meta_data';
+import {
+    getWethTokenFromTokensMetaDataByNetworkId,
+    mapTokensMetaDataToTokenByNetworkId,
+} from '../util/token_meta_data';
+import { Token } from '../util/types';
+
+export class KnownTokens {
+    private readonly _tokens: Token[] = [];
+    private readonly _wethToken: Token | null;
```

Comment:
Maybe I'm missing something here, but I think there's no need to make this nullable. And the app shouldn't work without the weth token anyway.